src/views/preview/preview.jsx

```diff
@@ -106,9 +111,27 @@ class Preview extends React.Component {
     }
     addEventListeners () {
         window.addEventListener('popstate', this.handlePopState);
+        window.addEventListener('orientationchange', this.setScreenFromOrientation);
     }
     removeEventListeners () {
         window.removeEventListener('popstate', this.handlePopState);
+        window.removeEventListener('orientationchange', this.setScreenFromOrientation);
+    }
+    setScreenFromOrientation () {
+        /*
+        * If the user is on a mobile device, switching to
+        * landscape format should make the fullscreen mode active
+        */
+        const isMobileDevice = screen.height <= frameless.mobile || screen.width <= frameless.mobile;
+        if (this.props.playerMode && isMobileDevice) {
+            const isLandscape = (screen.orientation.type && screen.orientation.type === 'landscape-primary') ||
```

Comment:
If `screen` or `screen.orientation` are `undefined` on browsers that don't support these, then this will cause a fatal error, won't it? I think you need to test for them (or just use `screen.height/width` for all).

Reply:
I wanted to be progressive but now I think just going with screen.width is the easiest option.